service/routes.py

```diff
@@ -133,13 +126,8 @@ def update_inventory_records(product_id):
     if not existing_record:
```

Comment:
we can call find_from_request_json and remove line 122,123,124

Reply:
Same as above.

Reply:
got it